contracts/PaymentChannels.sol

```diff
@@ -102,13 +102,13 @@ contract PaymentChannels is ECVerify, MintableToken {
     function incrementBalance(address _addr, uint _value)
         internal
     {
-        balances[_addr] = balances[_addr].add(_value);
+        ethBalances[_addr] = ethBalances[_addr].add(_value);
```

Comment:
Maybe this function should be in EthWallet

Reply:
Kinda bugs me that ethBalances is just kind of floating around

Reply:
Good idea, I'll add this change in the next PR